.golangci.toml

```diff
@@ -55,10 +55,10 @@
         "staticcheck",
         "stylecheck",
         "tagalign",
-        "tenv",
         "unconvert",
         "unparam",
         "unused",
+	"usetesting",
```

Comment:
```suggestion
        "usetesting",
```

Reply:
Well that's super weird, I manually typed 8 spaces and it converted it to tab? But it definitely looks misaligned currently.

Reply:
committed suggestion!